WalletWasabi/KeyManagement/KeyManager.cs

```diff
@@ -578,7 +578,7 @@ public ExtKey GetMasterExtKey(string password)
 			}
 			catch (SecurityException ex)
 			{
-				throw new SecurityException("Invalid password.", ex);
+				throw new SecurityException("The password is invalid.", ex);
```

Comment:
NACK, because changing exception messages are dangerous, I agree it'd be better like this though.

Reply:
Dangerous?! You have to explain that one ... ;-)

Reply:
While I am trying not to catch exceptions based on messages, I know I made this stupid mistake back then quite frequently. So if you change an exception message and I am expecting the old one, it'll fail. Also there are some exception message translations to human language into UI context, which could fail, because of changes like this.